packages/collector/test/tracing/cloud/aws-sdk/v3/kinesis/util.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const semver = require('semver');
+const minNodeJsVersion = '14.0.0';
+const { KinesisClient, DescribeStreamCommand, DeleteStreamCommand } = require('@aws-sdk/client-kinesis');
+
+if (semver.lt(process.versions.node, minNodeJsVersion)) {
+  exports.cleanup = async function () {
```

Comment:
I am still wondering why we need to check for the Node version in the cleanup script?
I know this was copied from s3, but it looks wrong. I might be wrong :)

Reply:
Removed the version check

Reply:
Great :)